StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -137,25 +149,41 @@ final class LinkAccountPickerViewController: UIViewController {
     }
 
     private func didSelectConectAccount() {
-        // TODO(kgaidis): implement repair bank account
-
-        guard let selectedAccount = dataSource.selectedAccount else {
+        guard let selectedAccountTuple = dataSource.selectedAccountTuple else {
```

Comment:
[Similar code in Stripe.js](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/dc01c2d95dfb34eb680dcdb7c29c6d14c43e2f18/src/linkedAccounts/inner/components/panes/AccountPicker/ReturningUser/ReturningUser.tsx#L208-L237)